charts/portal/ci/ci-values.yaml

```diff
@@ -1,2 +1,7 @@
+global:
```

Comment:
@Gazza7205 what is the benefit of adding the global.values when compared to values.yaml?

Reply:
ci-values are used in the CI actions